bin/rm.js

```diff
@@ -23,15 +25,8 @@ const target = process.argv[ 2 ];
 const extensions = process.argv.slice( 3 );
 
 const deleteFolderRecursive = ( path ) => {
-	fs.readdirSync( path ).forEach( ( file ) => {
-		const curPath = path + '/' + file;
-		if( fs.lstatSync( curPath ).isDirectory() ) {
```

Comment:
`if(` :rotating_light:

Reply:
D'Oh! Bad `lint` for not running here :)